### PR TITLE
VFS retention tests

### PR DESCRIPTION
**Pull request by lptr**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+    url = https://github.com/gradle/santa-tracker-android
 	branch = agp-3.6.0
 [submodule "santa-tracker-java"]
 	path = santa-tracker-java
-	url = git@github.com:gradle/santa-tracker-android.git
+	url = https://github.com/gradle/santa-tracker-android
 	branch = agp-3.6.0-java
+[submodule "gradle-profiler"]
+	path = gradle-profiler
+	url = https://github.com/gradle/gradle-profiler
```
Comment (wolfs) [inline .gitmodules:11]: I would prefer to use a released version of Gradle profiler.

Comment (lptr) [reply .gitmodules:11]: That would be ideal, but I think this is already a worthwhile step.

Comment (wolfs) [reply .gitmodules:11]: Releasing Gradle profiler is a one-click operation, I don't see why we should  build from source as a first step.

Comment (lptr) [reply .gitmodules:11]: Releasing is one part. The other is provisioning the pinned version. How would you do that here?

Comment (wolfs) [reply .gitmodules:11]: Using `curl` and `unzip`? And their counterparts on Windows?

Comment (lptr) [reply .gitmodules:11]: That's kind of my point. And then I have to test it on both Windows and *NIX. The submodule seemed simpler to me. You can also pin it to a released version of the profiler if you want.

There's also the upside of using a submodule that you can change `gradle-profiler` alongside the benchmark itself, which was a necessity several times. Maybe those times are over, and in any case, we could use a composite build, but... again, this seemed simpler.

All that said, I won't stop you if you want to make it work with released versions.

**Pull request merged**